From 2914bb3851aeff77daa2e86dabede3863dbfd855 Mon Sep 17 00:00:00 2001
From: Karolin Seeger <kseeger@samba.org>
Date: Fri, 29 Apr 2016 11:16:45 +0200
Subject: [PATCH 1049/1096] WHATSNEW: Udpate release notes.

Signed-off-by: Karolin Seeger <kseeger@samba.org>

Autobuild-User(v4-4-test): Karolin Seeger <kseeger@samba.org>
Autobuild-Date(v4-4-test): Sat Apr 30 16:35:06 CEST 2016 on sn-devel-144

(cherry picked from commit 0c535210dd565abfc3cc59eec15811af549d393b)
---
 WHATSNEW.txt | 15 +++++++++++++++
 1 file changed, 15 insertions(+)

diff --git a/WHATSNEW.txt b/WHATSNEW.txt
index 4a10fc5..88e324e 100644
--- a/WHATSNEW.txt
+++ b/WHATSNEW.txt
@@ -6,6 +6,10 @@
 
 This is the latest stable release of Samba 4.4.
 
+This release fixes some regressions introduced by the last security fixes.
+Please see bug https://bugzilla.samba.org/show_bug.cgi?id=11849 for a list of
+bugs addressing these regressions and more information.
+
 
 Changes since 4.4.2:
 --------------------
@@ -36,6 +40,14 @@ o  Volker Lendecke <vl@samba.org>
 
 o  Stefan Metzmacher <metze@samba.org>
    * BUG 11789: s3:wscript: pylibsmb depends on pycredentials.
+   * BUG 11841: Fix NT_STATUS_ACCESS_DENIED when accessing Windows public share.
+   * BUG 11847: Only validate MIC if "map to guest" is not being used.
+   * BUG 11849: auth/ntlmssp: Add ntlmssp_{client,server}:force_old_spnego
+     option for testing.
+   * BUG 11850: NetAPP SMB servers don't negotiate NTLMSSP_SIGN.
+   * BUG 11858: Allow anonymous smb connections.
+   * BUG 11870: Fix ads_sasl_spnego_gensec_bind(KRB5).
+   * BUG 11872: Fix 'wbinfo -u' and 'net ads search'.
 
 o  Tom Mortensen <tomm@lime-technology.com>
    * BUG 11875: nss_wins: Fix the hostent setup.
@@ -56,6 +68,9 @@ o  Uri Simchoni <uri@samba.org>
    * BUG 11815: smbcquotas: print "NO LIMIT" only if returned quota value is 0.
    * BUG 11852: libads: Record session expiry for spnego sasl binds.
 
+o  Hemanth Thummala <hemanth.thummala@nutanix.com>
+   * BUG 11840: Mask general purpose signals for notifyd.
+
 
 #######################################
 Reporting bugs & Development Discussion
-- 
2.9.0

